docker/jmeter-master/launcher.sh

```diff
@@ -13,13 +13,17 @@ run_jmeter_test() {
     cat output.log
     if grep "end of run" ./output.log; then
       echo "=== Jmeter is finished! ==="
-      echo "=== Trying to send report to ${AWS_BUCKET_NAME}/${LOADTEST_NAME} endpoint ${AWS_ENDPOINT_URL} ==="
-      if [[ -n "${AWS_BUCKET_NAME}" ]]; then
+      if [[ -n "${REPORT_PRESIGNED_URL}" ]]; then
         echo "=== Saving report to Object storage ==="
+        tar -C /results -cf results.tar .
+        curl -X PUT -H "Content-Type: application/x-tar" -T results.tar -L "${REPORT_PRESIGNED_URL}"
+      elif [[ -n "${AWS_BUCKET_NAME}" ]]; then
```

Comment:
Shall we check here if `AWS_BUCKET_NAME` not empty string instead?

Reply:
you mean check it before checking for `REPORT_PRESIGNED_URL` ?
If is what you mean, I would prefer to check for `REPORT_PRESIGNED_URL` first, as all `AWS_*` envs are deprecated